docs/_docs/01-quick-start-guide.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Quick-Start Guide"
+permalink: /docs/quick-start-guide/
+excerpt: "How to quickly install and setup Minimal Mistakes for use with GitHub Pages."
+last_modified_at: 2024-05-05T08:48:05-04:00
+redirect_from:
+  - /theme-setup/
+toc: true
+---
+
+
+**Note:** BeSLab only suported on Ubuntu based system.

```

Comment:
At present its supporting only Ubuntu. May be we can say it at present only Ubuntu with version 22.04 specifically.